bin/utils/getconfig.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+#  Based on Texas Instruments Incorporated - http://www.ti.com/
+
+
+import logging
+import numpy as np
+import os
+import sys
+import yaml
+
+
+def read_file(model_dir):
+    try:
+        with open(model_dir + "param.yaml", "r") as file:
+            return yaml.safe_load(file)
+    except OSError as e:
+        logging.error("YAML file not found")
+        sys.exit(1)
```

Comment:
Never exit like this. Return an error, or an exception, and let the client decide if they want to exit.